Clean/Circuit/Expression.lean

```diff
@@ -15,6 +15,30 @@ inductive Expression (F : Type) where
   | add : Expression F -> Expression F -> Expression F
   | mul : Expression F -> Expression F -> Expression F
 
+
+partial def exprToJson : Expression F → Lean.Json
+  | .var v   => Lean.Json.mkObj [
+                  ("type", Lean.Json.str "var"),
+                  ("index",     Lean.Json.num v.index)   -- Nat’s instance
+                ]
+  | .const c => Lean.Json.mkObj [
+                  ("type", Lean.Json.str "const"),
+                  ("value", "cst")   -- F’s instance
```

Comment:
I think the correct way to do this is assume `[Lean.ToJson F]` as a parameter to this function, and then just serialize the constant `c` using `Lean.ToJson.toJson c`

Reply:
Got it. Fixed